api/models/fixtures/cron_labels.yml

```diff
@@ -7,4 +7,4 @@ main:
 really-long-process-type-name:
   build: .
   labels:
-    - convox.cron.really-long-cron-job-name=0 * * * ? bin/myjob
```

Comment:
@ddollar We specify that dashes are not valid in cron names.

https://convox.com/docs/scheduled-tasks/ 

> Task names may not be reused within the same application process and must be alphanumeric

The reason we did that is because if we stripped dashes you could have a job called `MyJob` and one called `my-job` and they would conflict. Now that I think about it, though, that same problem can happen with processes.

`my-process` and `my_process` could currently conflict. Will think about this some more . . .